Script Analysis Examples/Test Data Analysis/Failure_Pareto_Result_Analysis.ipynb

```diff
@@ -0,0 +1,642 @@
+    "            - `y`: The dataframe column corresponding to the y-axis values.\n",
+    "            - `style`: The plot's style. Accepted values are ['LINE', 'BAR', 'SCATTER'].\n",
+    "            - `color`: The plot's color. Accepted formats are ['blue', '#0000ff', 'rbg(0,0,255)'].\n",
+    "            - `label`: The plot's name, to be shown in a plot legend. \n",
+    "            - `secondary_y`: Whether or not to display this plot on a second y-axis.\n",
+    "            - `GROUP_BY`: A list of columns in the dataframe on which to group data, e.g. to color individual points.\n",
+    "        - `orientation`: 'HORIZONTAL' or 'VERTICAL'.\n",
+    "        - `stacked`: Whether or not to display the plots stacked on top of each other.\n",
+    "\n",
+    "Here is an example of a notebook result with two outputs, one of which is a dataframe with two columns, and the other is a scalar value:\n",
+    "```\n",
+    "[{\n",
+    "    'type': 'data_frame',\n",
+    "    'id': 'output_id_1',\n",
+    "    'data': {\n",
+    "        'columns': [\n",
+    "            {'name': 'time', 'type': 'datetime'},\n",
+    "            {'name': 'value', 'type': 'number'}\n",
+    "         ],\n",
+    "        'values': [\n",
+    "            ['2020-09-29T00:00:00.000Z', 46.1538461538],\n",
+    "            ['2020-09-30T00:00:00.000Z', 63.1578947368],\n",
+    "            ...\n",
+    "         ]\n",
+    "    },\n",
+    "    'config': {\n",
+    "        'title': 'My Title',\n",
+    "        'graph': {\n",
+    "            'axis_labels': ['X Axis', 'Y Axis'],\n",
+    "            'orientation': 'VERTICAL',\n",
+    "            'plots': [\n",
+    "                {'x': 'time', 'y': 'value', 'style': 'BAR', 'color': '#0000ff', 'label': 'Plot 1'}\n",
+    "            ]\n",
+    "        }\n",
+    "    }\n",
+    "}, {\n",
+    "    'type': 'scalar',\n",
+    "    'id': 'output_id_2',\n",
+    "    'config': {\n",
+    "        'title': 'My Title'\n",
+    "    },\n",
+    "    'value': 5\n",
+    "}]\n",
+    "```\n",
+    "\n",
+    "For this report, there is one output, which is a dataframe with three columns. The first column contains the group categories, which are part numbers by default. The second column contains failure counts, and the third column contains the cumulative totals as percents.\n",
+    "\n",
+    "| part_number | fail_count | cumulative |\n",
+    "|-------------|------------|------------|\n",
+    "| 151837H     | 102        | 34.459459  |\n",
+    "| 154261F     | 98         | 67.567568  |\n",
+    "| 193343E     | 96         | 100        |\n",
+    "\n",
+    "The graph configuration specifies two plots: A bar chart for the failure counts and a line chart for the cumulative percentage. We use Pandas to convert the dataframe built in the previous cells into a tabular format and then return that with the result object."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
We dont' need this cell either. 

We should make sure that the returned values through scrapbook tell something about the generated png.

Reply:
updated to remove the json output and mentioned about the generated png uploaded to file service.